notifications/04-02-20-0600.md

```diff
@@ -0,0 +1,7 @@
+**RFC192830 CITZ - PROD MCS - PRB0040383 Change Load Balancer to use all three infra nodes**
+
+Thursday April 2 @ 06:00 AdvSol will be working with the network vendor to update the load balancer config for `*.pathfinder.gov.bc.ca:80/443 - (142.34.208.209)`. We expect a 30 second interruption to traffic as the config is applied to the load balancer. @steven.barre will be up and monitoring the traffic flows.
+
+Currently the load balancer forwards all traffic to one IP `*.pathfinder.bcgov:80/443 (142.34.143.180)` which is moved between the infra nodes as needed during maintenance. This causes all the traffic on the platform to go through only one infra node at a time, and we apparently reached the scalability limit of that last week. After the change the traffic will forward to all 3 infra nodes and spread the load out.
+
+Additionally, we would ask that if your project is still relying on `*.pathfinder.bcgov:80/443 (142.34.143.180)` that you transition to the external LB `*.pathfinder.gov.bc.ca:80/443 - (142.34.208.209)` for improved stability of your traffic.

```

Comment:
In addition to improved stability, perhaps a note about access from external networks would be wise as well? Seeing as we're all supposed to be responsible with our use of VPN right now, and moving services from `*.pathfinder.bcgov:80/443` to `*.pathfinder.gov.bc.ca:80/443` removes the need to hop onto VPN just for DNS.

Reply:
Too slow @wenzowski , its already been posted to #devops-alerts. Good points though!